RatTracker-WPF/RatTracker-WPF/CmdrJournalParser.cs

```diff
@@ -426,6 +426,8 @@ public string[] WriteSafeReadAllLines(string path)
           file.Add(sr.ReadLine());
         }
 
+        sr.Close();
```

Comment:
Closing is unnecessary as the using directive auto closes the StreamReader when leaving the block.